nx.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
What if we dnt want to use lerna cache feature at all for now? can we simply remove this ?